From: Ichthyostega <prg@ichthyostega.de>
Date: Sun, 9 Aug 2015 16:09:04 +0200
Subject: Build against Lua from system

build against Lua installed on the system,
instead of Lua interpreter shipped with the package
---
 Client/Makefile.am           | 1 -
 MSVC/SettingsGUI/Makefile.am | 3 +--
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/Client/Makefile.am b/Client/Makefile.am
index a4983e7..50a4850 100644
--- a/Client/Makefile.am
+++ b/Client/Makefile.am
@@ -21,7 +21,6 @@ AM_CPPFLAGS = \
 	-I ../ContentDecoder \
 	-I ../TupleStorage \
 	-I ../ContentDownloader \
-	-I ../lua5.1/src \
 	-I ../Client \
 	-I ../Networking \
 	-I ../tinyXml
diff --git a/MSVC/SettingsGUI/Makefile.am b/MSVC/SettingsGUI/Makefile.am
index a604339..de17af6 100644
--- a/MSVC/SettingsGUI/Makefile.am
+++ b/MSVC/SettingsGUI/Makefile.am
@@ -19,7 +19,6 @@ AM_CPPFLAGS = \
 	-I ../../ContentDecoder \
 	-I ../../TupleStorage \
 	-I ../../ContentDownloader \
-	-I ../../lua5.1/src \
 	-I ../../Client \
 	-I ../../Networking \
 	-I ../../tinyXml \
@@ -53,7 +52,7 @@ electricsheep_preferences_SOURCES = \
 electricsheep_preferences_LDADD = -lboost_thread -lboost_filesystem $(CURL_LIBS) $(LUA_LIBS) -ltinyxml @WX_LDADD@ @BOOST_LDADD@
 
 #AM_CXXFLAGS = $(linux_CFLAGS) -fpermissive @WX_CXXFLAGS@
-AM_CXXFLAGS = $(linux_CFLAGS) @WX_CXXFLAGS@
+AM_CXXFLAGS = $(linux_CFLAGS) $(LUA_CFLAGS) @WX_CXXFLAGS@
 
 
 
